libzeth/tests/prover/prover_test.cpp

```diff
@@ -91,22 +97,22 @@ bool TestValidJS2In2Case1(
             "AAAA00000000000000000000000000000000000000000000000000000000EEE"
             "E"),
         trap_r_bits256);
-    joinsplit_input<FieldT, TreeDepth> input(
+    joinsplit_input<FieldT, tree_depth> input(
```

Comment:
We can polish the style of template params with: `readability-identifier-naming.TemplateParameterCase` and set the check option key `readability-identifier-naming.TemplateParameterSuffix` to `T`